src/main/scala/edu/ie3/simona/model/participant/load/random/RandomLoadModel.scala

```diff
@@ -146,6 +145,7 @@ final case class RandomLoadModel(
       case Some(foundIt) => foundIt
       case None          =>
         /* Instantiate new gev distribution, put it to storage and return it */
+        val randomFactory = RandomFactory.get((Math.random() * 1000000).toInt)
```

Comment:
We can implement a 
`val randomFactory: RandomFactory = RandomFactory.get(42L)`
in the companion object so we don't need to set up individual factories for every load model